### ISSUE TITLE
Add geojson data to some webmap framework

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/census2020#39]
- Leaflet, Mapbox
- some React based wrapper
- first pass at this knowing we don't have much product definition on this task yet.

**Comment by avickers** [comment]
https://react-leaflet.js.org/docs/en/intro.html
https://www.gatsbyjs.org/packages/gatsby-plugin-react-leaflet/

I have moderate experience with Leaflet, minimal experience with React, and no experience with either Gatsby or the USWDS.  I also have two novice developers that I am going to help teach how to create interactive maps.

I think that I'll take a two pronged approach: A personal library as a React-Leaflet sandbox; and then, a branch from this repo for integration of the four parts.

I can use the Sandbox for teaching and prototyping mapping, and thus; avoid needing to explain either Gatsby or the USWDS, which is especially beneficial considering that I don't actually know anything about either of them.

With luck, it won't prove challenging to integrate the React-Leaflet components into a Gatsby environment.  That's the whole point of components anyway, right?

It's a bold strategy, Cotton.  Let's see if it pays off for me.

**Comment by avickers** [comment]
So the geoJSON data isn't precisely right.  The ESRI shapefiles the CoA produces don't use EPSG:4326 the CRS (Coordinate Reference System) that GPS satellites and most tech companies use.

I don't know what CRS the shapefiles are using; however, after recursively remapping the coordinates to fit on a map in the general vicinity of Texas, I observed they were tilted on an axis; and, that finally clued me in to fact that the trouble wasn't either the conversion or the shapefiles themselves but rather the CRS being used.

I then had to go figure out which CRS Leaflet expected (EPSG:4326) and a way to convert the Shapefiles to it.  

Fun fact, Leaflet actually converts EPSG:4326 to Web Mercator (EPSG 3857) before painting.
